tests/db/model/collector/methods.js

```diff
@@ -0,0 +1,257 @@
+      });
+    });
+
+    describe('findAliveCollector', () => {
+      it('some over threshold', () => {
+        const threshold = 3000;
+        const fakeNow = new Date('2018-05-22T14:51:07');
+        clock = sinon.useFakeTimers(fakeNow);
+        collectorConfig.heartbeatLatencyToleranceMillis = threshold;
+
+        return Collector.findAliveCollector()
+        .should.eventually.have.property('name', collector3.name);
+      });
+
+      it('all over threshold', () => {
+        const threshold = 1000;
+        const fakeNow = new Date('2018-05-22T14:51:07');
+        clock = sinon.useFakeTimers(fakeNow);
+        collectorConfig.heartbeatLatencyToleranceMillis = threshold;
+
+        return Collector.findAliveCollector()
+        .should.eventually.not.exist;
+      });
+
+      it('none over threshold', () => {
+        const threshold = 10000;
+        const fakeNow = new Date('2018-05-22T14:51:07');
+        clock = sinon.useFakeTimers(fakeNow);
+        collectorConfig.heartbeatLatencyToleranceMillis = threshold;
+
+        return Collector.findAliveCollector()
+        .should.eventually.have.property('name', collector2.name);
+      });
+    });
+
+    describe('checkMissedHeartbeat', () => {
+
+      beforeEach(() => Promise.join(
+        GeneratorTemplate.upsert(generatorTemplate),
+        Generator.upsert(generator1),
+        Generator.upsert(generator2),
+        Generator.upsert(generator3),
+      ));
+
+      afterEach(u.forceDelete);
+
+      it('checkMissedHeartbeat', () => {
+        const threshold = 3000;
+        const fakeNow = new Date('2018-05-22T14:51:07');
+        clock = sinon.useFakeTimers(fakeNow);
+        collectorConfig.heartbeatLatencyToleranceMillis = threshold;
+
+        return Collector.checkMissedHeartbeat()
+        .then(() => Promise.join(
+          Generator.find({ where: { name: generator1.name } }),
+          Generator.find({ where: { name: generator2.name } }),
+          Generator.find({ where: { name: generator3.name } }),
+          Collector.find({ where: { name: collector2.name } }),
+        ))
+        .spread((gen1, gen2, gen3, coll2) => {
+          expect(gen1.currentCollector).to.equal(collector1.name);
+          expect(gen2.currentCollector).to.equal(collector3.name);
+          expect(gen3.currentCollector).to.equal(collector3.name);
+          expect(coll2.status).to.equal(collectorStatuses.MissedHeartbeat);
+        });
+      });
+    });
+  });
+
+  describe('instanceMethods', () => {
+    describe('isRunning', () => {
+
+      it('running', () => {
+        Collector.build({ status: collectorStatuses.Running })
+        .isRunning().should.be.true;
+      });
+
+      it('not running', () => {
+        Collector.build({ status: collectorStatuses.Stopped })
+        .isRunning().should.be.false;
+      });
+
+      it('undefined', () => {
+        Collector.build({})
+        .isRunning().should.be.false;
+      });
+
+    });
+
+    describe('isAlive', () => {
```

Comment:
easier to read the test results when every describe ends with ` >`, e.g. `describe('isAlive >'), () => {`